.github/workflows/build.yml

```
@@ -186,10 +186,10 @@ jobs:
           toolchain: stable
           override: true
 
-      - run: python -m pip install tox
+      - run: python -m pip install maturin
         working-directory: ./bindings/python
 
-      - run: tox -e build-sdist
+      - run: maturin build --sdist -o dist
```

This currently builds both sdist and a wheel, to only build sdist:

```suggestion
      - run: maturin sdist -o dist
```

---

Oh, right! Thanks